apps/examples/testcase/le_tc/tcp_tls/Kconfig

```diff
@@ -0,0 +1,57 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_TCP_TLS_STRESS 
+	bool "TCP TLS Strss Test Example"
+	default n
+	select TLS_TCP_STRESS
+	depends on STRESS_TOOL
```

Comment:
I think 'NET_SECURITY_TLS' should be enabled to use this TC.
This needs to have the dependency of NET_SECURITY_TLS.

Reply:
That is a good point. Thanks for the comment.
Modification completed.